packages/contracts-bedrock/src/L2/OptimismSuperchainERC20Beacon.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import { IBeacon } from "@openzeppelin/contracts-v5/proxy/beacon/IBeacon.sol";
+import { ISemver } from "src/universal/ISemver.sol";
+
+/// @custom:proxied
```

Comment:
This contract isn't really proxied, right?

Reply:
All predeploys are proxied by default, maybe we should add it to the not proxied list

Reply:
If this the case, then leaving it like this is okay. I had the idea that this one was just going to have different code installed on hardforks, but I really don't remember where I got that from.